testing/tcc/APKBUILD

```diff
@@ -0,0 +1,34 @@
+# Contributor:
+# Maintainer: Hinata Yanagi <hinasssan@gmail.com>
+pkgname=tcc
+pkgver=2017.10.19
+pkgrel=0
+pkgdesc="Tiny C Compiler"
+url="http://repo.or.cz/tinycc.git"
+arch="all"
+license="the GNU Lesser General Public License"
+depends="musl-dev"
```

Comment:
other compilers such as gcc or pcc don't depend on musl-dev either, I would leave it up to the user to install it.